content/basics.tex

```diff
@@ -7,3 +7,30 @@ \subsection{Compiling with sanitize options}
 \begin{lstlisting}[breakatwhitespace=true]
 g++ -std=gnu++17 -g -fsanitize=address -fsanitize=undefined -fno-sanitize-recover file.cpp -o file
 \end{lstlisting}
+
+\subsection{Stress test}
```

Comment:
I'd move this section to the end of the booklet, since it's not something we're going to use in all contests or at the start of one. We could add a `misc` section containing this code and potentially more in the future